core/src/components/chip/chip.ios.scss

```diff
@@ -0,0 +1,26 @@
+@import "./chip";
+@import "./chip.ios.vars";
+
+:host {
+  /**
+   * Main content should be prioritized on iOS,
+   * so we set max font size for chips.
+   * We add a min font size to keep the text legible.
+   */
+  font-size: $chip-ios-font-size;
+}
+
+// Chip Avatar
+// ---------------------------------------------
+
+::slotted(ion-avatar) {
+  width: calc($chip-ios-font-size * 24 / 14);
+  height: calc($chip-ios-font-size * 24 / 14);
+}
+
+// Chip Icon
+// ---------------------------------------------
+
+::slotted(ion-icon) {
+  font-size: calc($chip-ios-font-size * 20 / 14);
```

Comment:
What do these 24 and 14 values represent?

Reply:
this is essentially providing a min & max size to the icon & avatar (for ios mode only). 
24 is the avatar's original width & height; 20 is the icon's original font size, and 14 is the font-size of the chip label when it's not being scaled. so here we're providing it a max & min value when the chip label font size has one.

i suppose i can convert these to say

```
::slotted(ion-avatar) {
  width: clamp(22px, 1.5rem, 38px);
  height: clamp(22px, 1.5rem, 38px);
}

::slotted(ion-icon) {
  font-size: clamp(18px, 1.25rem, 32px);
}
```

which does approximately the same thing. but these numbers feel more "magical" and less precise to me than the numbers i had before, which were just the original sizes.

Reply:
also, i couldn't find anywhere in the design doc or anywhere else where we should get the max & min ios font sizes from. so my earlier code had just two "made up" numbers, whereas this version has 6 of them.